source/_integrations/torque.markdown

```diff
@@ -29,7 +29,7 @@ Under the **Logging Preferences** header:
 Under the **Realtime Web Upload** header:
 
 - Check **Upload to web-server**.
-- Enter `https://HOST:PORT/api/torque` as the **Web-server URL**, where `HOST` and `PORT` are your externally accessible Home Assistant HTTP host. To use a Bearer Token, this has to be [SSL/TSL](/docs/ecosystem/certificates/).
+- Enter `https://HOST/api/torque` or `https://@/HOST:PORT/api/torque` as the **Web-server URL**, where `HOST` and `PORT` are your externally accessible Home Assistant HTTP host. To use a Bearer Token, this has to be [SSL/TSL](/docs/ecosystem/certificates/).
```

Comment:
I'm not sure what the added `@/` means, could you explain that?

Reply:
The way the torque app works with the url is that it takes the host, and the endpoint, and put them together, ignoring the ip, so that `https://HOST:PORT/api/torque` becomes just `https://HOST/api/torque`. We dont want that, so the @ fools the app into thinking that the `HOST:PORT/api/torque` is the whole host, thus, keeping the port.

I decompiled the app and recriated the code that is used to send the info to the webserver, and messed with the url until it worked.

In this image you can see how it is implemented, and the result of the normal url and the modified one.

![image](https://user-images.githubusercontent.com/19365140/194786481-20c40ea4-d2cd-432b-a17d-9270b52dee81.png)

Reply:
Thanks for providing the context 👍